lib/auth/auth.go

```diff
@@ -3923,17 +3923,21 @@ func (a *Server) DeleteNamespace(namespace string) error {
 	}
 	return a.Services.DeleteNamespace(namespace)
 }
-
 func (a *Server) CreateAccessRequest(ctx context.Context, req types.AccessRequest, identity tlsca.Identity) error {
+	_, err := a.CreateAccessRequestV2(ctx, req, identity)
+	return trace.Wrap(err)
+}
+
+func (a *Server) CreateAccessRequestV2(ctx context.Context, req types.AccessRequest, identity tlsca.Identity) (types.AccessRequest, error) {
```

Comment:
This is only an internal function call. I introduced the new V2 API but currently will be only used here https://github.com/gravitational/teleport.e/pull/1827 only when a dry-run (new argument) is passed. We want to merge these changes in v13.3, so we cannot expect that all auths/proxies will be upgraded to the latest. Because of that, I don't want to use the new V2 API anywhere as default.